app/presenters/image_upload_response_presenter.rb

```diff
@@ -78,6 +78,7 @@ def attention_with_barcode?
   end
 
   def ocr_pii
```

Comment:
I thought we only did anything with this if it was unsuccessful?

Noting how the use of it...

https://github.com/18F/identity-idp/blob/4926cdd948f3db77d98e3360fc3ec0bd2757be52/app/javascript/packages/document-capture/services/upload.ts#L119-L121

...is within a block to check for unsuccessful:

https://github.com/18F/identity-idp/blob/4926cdd948f3db77d98e3360fc3ec0bd2757be52/app/javascript/packages/document-capture/services/upload.ts#L102

Reply:
great point ... for the BarcodeAttentionWarning screen to appear,  the barcode attention should be the [only reason why the result is failed](https://github.com/18F/identity-idp/blob/cd62b60de30972e7d272e2d741fec862f2098aef/app/presenters/image_upload_response_presenter.rb#L36)